src/igvfd/audit/file.py

```diff
@@ -112,3 +112,34 @@ def audit_file_no_file_format_specifications(value, system):
             f'has no `file_format_specifications`.'
         )
         yield AuditFailure(audit_message.get('audit_category', ''), f'{detail} {audit_message.get("audit_description", "")}', level=audit_message.get('audit_level', ''))
+
+
+@audit_checker('IndexFile', frame='object')
+@audit_checker('AlignmentFile', frame='object')
+@audit_checker('TabularFile', frame='object')
+@audit_checker('ReferenceFile', frame='object')
+@audit_checker('ModelFile', frame='object')
+@audit_checker('SequenceFile', frame='object')
+def audit_file_missing_anvil_url(value, system):
+    '''
+    [
+        {
+            "audit_description": "Released and archived files that are restricted access are expected to be loaded in anvil.",
+            "audit_category": "missing anvil url",
+            "audit_level": "INTERNAL_ACTION"
+        }
+    ]
+    '''
+    object_type = space_in_words(value['@type'][0]).capitalize()
+    audit_message = get_audit_message(audit_file_missing_anvil_url, index=0)
+    if value.get('status') in ['released', 'archived'] and value.get('controlled_access', False) and not (value.get('anvil_url')):
+        file_set_object = system.get('request').embed(
+            value.get('file_set'), '@@object_with_select_calculated_properties?field=data_use_limitation_summaries')
+        if file_set_object.get('status') in ['released', 'archived']:
+            data_use_limitation_summaries = file_set_object.get('data_use_limitation_summaries', '')
+            if data_use_limitation_summaries and data_use_limitation_summaries != ['no certificate']:
```

Comment:
Controlled-access file but no certificates? Seems a little shaky to rely on string value from summary.

Reply:
There are a series of audits that ensure a certificate is submitted for file sets that require one prior to the release of either the file or file set so this will likely never be encountered.

Reply:
@keenangraham I think the way to think about the "no certificate" state is that if we have said that the file is controlled, but a certificate does not exist, we don't want to put this into anvil yet as something is wrong